.deepsource.toml

```diff
@@ -0,0 +1,24 @@
+version = 1
+
+test_patterns = [
+  "tests/**",
+  "test_*.py"
+]
+
+exclude_patterns = [
+  "static/**",
+  "staticfiles/**",
```

Comment:
@camilamaia why are there two static files in the project ?

Reply:
@Pradhvan good question, I was trying to make it work, so it ended up like this 🤷‍♀️ I'm kinda noob with this static config. Should we have just one? How can we fix it?

Reply:
@camilamaia let's create a separate issue for it.